### ISSUE TITLE
Model Development

### ISSUE THREAD

**Issue by SLuckLeonard** [SLuckLeonard/Image-Classification-using-CNN#2]
TensorFlow 2-3 convolutional layers followed by pooling layers and fully connected layers.

Deliverables: Python script with the CNN architecture.

Data augmentation integrated with the training loop.

Script showing the compiled model ready for training.

**Comment by SLuckLeonard** [comment]
Finished initial model training with an accuracy of .65. Looking for higher accuracy, will address with model evaluation.
